app/views/admin/admin_users/_user_history.html.erb

```diff
@@ -24,6 +26,28 @@
       <td><%= ts("Account Created") %></td>
       <td><%= ts("System Generated") %></td>
     </tr>
+    <tr>
+      <td>
+        <% if @user.current_sign_in_at.nil? %>
+          <%= ts('No logins recorded') %>
+        <% else %>
+          <%= @user.current_sign_in_at %> ( <%= @user.current_sign_in_ip %> )
```

Comment:
You should remove spaces inside parentheses. We don't do that in code or in [user text](https://github.com/otwcode/otwarchive/blob/dd6ca1180f875ae8def2b21b7b78adc68c805c1f/app/views/subscriptions/index.html.erb#L43).